airbyte-integrations/connectors/destination-snowflake/src/test/java/io/airbyte/integrations/destination/snowflake/SnowflakeDestinationTest.java

```diff
@@ -74,18 +73,6 @@ void testUrlPattern(final String url, final boolean isMatch) throws Exception {
     assertEquals(isMatch, matcher.find());
   }
 
-  @ParameterizedTest
-  @MethodSource("destinationTypeToConfig")
-  public void testS3ConfigType(final String configFileName, final DestinationType expectedDestinationType) throws Exception {
```

Comment:
doublechecking: this test was important for when we supported s3/gcs staging, but we deleted those so now we can delete the test?

Reply:
Yeah. something something SwitchingDestination.. destinationType check. which is always InternalStaging